### ISSUE TITLE
Why controls always hidden ?

### ISSUE THREAD

**Issue by fznhq** [6c65726f79/custom-titlebar#67]
I tried this plugin with electron 16.0.2 and when i try using example here, class `.custom-titlebar-controls` always got `visibilty: hidden` from the start, like this:
![controls-hidden](https://user-images.githubusercontent.com/55099255/146167896-541e83b6-3a68-4588-a545-a6235787c5c2.PNG)
my solution right now is just to add `visibility: visible !important` to `.custom-titlebar-controls`, but it needs to do that manually, or I'm doing something wrong? my code:
```javascript
const Titlebar = require('@6c65726f79/custom-titlebar');
const { platform } = require('process');
new Titlebar({
        backgroundColor: '#37474f',
        platform: platform,
        icon: 'logo.ico',
        titleHorizontalAlignment: 'left',
})
```

**Comment by 6c65726f79** [comment]
The controls should be hidden only when the Window Controls Overlay is enable. There may be a problem with the detection, I'll fix that today. Thank you for reporting the issue.

**Comment by fznhq** [comment]
After reading electron [documentation](https://www.electronjs.org/docs/latest/tutorial/window-customization#window-controls-overlay-macos-windows), turn out `navigator.windowControlsOverlay` need to be activated by using
```javascript
new BrowserWindow({ titleBarOverlay: true })
```
so yeah basically apps that have not used that feature cant even show the control because `navigator.windowControlsOverlay` is not even available

**Comment by 6c65726f79** [comment]
Yes, that was the problem, `navigator.windowControlsOverlay.visible` was returning `undefined` instead of `false`. This is fixed in v0.5.2, thank you again.
